torch/_inductor/codegen/triton.py

```diff
@@ -80,6 +80,12 @@ def _print_floor(self, expr):
         assert len(expr.args) == 1
         return f"tl.libdevice.floor({self.paren(self._print(expr.args[0]))})"
 
+    def _print_ceiling(self, expr):
+        assert len(expr.args) == 1
+        return (
+            f"tl.libdevice.ceil({self.paren(self._print(expr.args[0]))}).to(tl.int64)"
+        )
```

Comment:
Used with floordiv, which doesn't support non-int args in Triton (see `assert False` in `floordiv` in `triton/language/semantic.py`).

Example expr:
```
  tmp0 = 4*((x0 // (tl.libdevice.ceil(((-3.0/4.0) + ((1.0/4.0)*ks1))))))

```

Reply:
This is oddly asymmetric with the floor above, which doesn't convert to int64. Why the int64 conversion?

Reply:
I think this is due to where these might show up based on our tests. This ceil expr is passed to floor. If floor in Triton handled non-int args, this wouldn't be necessary. Note that I also have to cast to int type on the C++ side, so at least that's consistent.